sql/core/src/test/scala/org/apache/spark/sql/parquet/ParquetIOSuite.scala

```diff
@@ -383,6 +385,8 @@ class ParquetIOSuiteBase extends QueryTest with ParquetTest {
   }
 
   test("SPARK-6352 DirectParquetOutputCommitter") {
+    val clonedConf = configuration
```

Comment:
Seems we have `protected def configuration = sparkContext.hadoopConfiguration` in `SQLTestUtils` and we are not cloning it at here?